Source/WebCore/Modules/credentialmanagement/CredentialsContainer.h

```diff
@@ -66,9 +68,8 @@ class CredentialsContainer : public RefCounted<CredentialsContainer> {
 private:
     ScopeAndCrossOriginParent scopeAndCrossOriginParent() const;
 
-    WeakPtr<Document, WeakPtrImplWithEventTargetData> m_document;
```

Comment:
Can we add an accessor method instead of moving this?

Reply:
Yeah, I was thinking the same. 

I wanted to land these two PRs one first, as it has the same issue:
https://github.com/WebKit/WebKit/pull/30499/files#diff-471d276e474722f3c5d2a47188f16f9cf6603d3c0aab4ee3d043e01d9c55cfe3

And then add the accessor as a followup just to avoid the merge conflicts. 

If it's ok with you, I'll do that as follow up straight after this one.

Reply:
Added the getter, as I ended up adding it to the other PR.